docs/resources/cbr_vault.md

```diff
@@ -220,8 +220,8 @@ The following arguments are supported:
 
 * `bind_rules` - (Optional, Map) Specifies the tags to filter resources for automatic association with **auto_bind**.
 
-* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the ID of the enterprise project to which the vault
-  belongs. Changing this will create a new vault.
+* `enterprise_project_id` - (Optional, String) Specifies the ID of the enterprise project to which the vault
+  belongs.
```

Comment:
```suggestion
* `enterprise_project_id` - (Optional, String) Specifies the ID of the enterprise project to which the vault belongs.
```
ensure that the text is neat during modification.

Reply:
ok, I will pay attention to it next.